docs/topics/Opcodes_and_Status_Codes.md

```diff
@@ -167,7 +167,10 @@ Along with the HTTP error code, our API can also return more detailed error code
 | 50034  | A message provided was too old to bulk delete                                                                                 |
 | 50035  | Invalid form body (returned for both `application/json` and `multipart/form-data` bodies), or invalid `Content-Type` provided |
 | 50036  | An invite was accepted to a guild the application's bot is not in                                                             |
-| 50041  | Invalid API version provided                                                                                                  |
+| 50041  | Invalid API version provided                
+|
+| 60003  | Two factor is required for this operation
```

Comment:
Minor nitpick; I'd probably refer to this as `2FA`, or `two-factor authentication` just to prevent confusion with what "two-factor" refers to. I realise these correspond to the messages provided iirc.